app/api/server/v1/channels.js

```diff
@@ -359,6 +360,12 @@ API.v1.addRoute('channels.history', { authRequired: true }, {
 			offset = parseInt(this.queryParams.offset);
 		}
 
+		const excludeTypes = [];
```

Comment:
New feature?

Reply:
yes! this param (and some refactors that preceede it) allows api consumers to filter out certain messages. it will be useful later when we implement the feature to hide system messages on the server (mobile clients currently have to do it on the front end)

Reply:
I'd suggest moving this to another PR then